src/Sentry.Unity.Editor.iOS/BuildPostProcess.cs

```diff
@@ -0,0 +1,43 @@
+using Sentry.Extensibility;
+using UnityEditor;
+using UnityEditor.Callbacks;
+
+namespace Sentry.Unity.Editor.iOS
+{
+    public static class BuildPostProcess
+    {
+        [PostProcessBuild(1)]
+        public static void OnPostProcessBuild(BuildTarget target, string pathToProject)
+        {
+            if (target != BuildTarget.iOS)
+            {
+                return;
+            }
+
+            // TODO: check for other criteria why we would stop touching the Xcode project
+            var options = ScriptableSentryUnityOptions.LoadSentryUnityOptions();
+            if (options is null)
+            {
+                return;
+            }
```

Comment:
I'll have to double check the "not calling twice". But it should (tm) work fine since we're using it 1x during runtime when we want to init and anytime else would be in the editor during building.